packages/common/util.ts

```diff
@@ -6,3 +6,20 @@ export type WithoutId<T extends {}> = Omit<T, 'id'>
 export type WithId<T extends {}> = T & {id: string}
 
 export const wait = (ms: number) => new Promise(resolve => setTimeout(resolve, ms))
+
+/** phone numbers must be E164 for Twilio: https://www.twilio.com/docs/glossary/what-e164 */
+const regex = /^\+1\d{10}(x\d+)?$/
+export const e164 = (number: string): string => {
```

Comment:
We don't need `e164` to return.  Instead, just have it return `null` or throw and call this `checkE164`.